src/main/java/org/broadinstitute/consent/http/service/ElectionService.java

```diff
@@ -189,6 +193,16 @@ public Election submitFinalAccessVoteDataRequestElection(Integer electionId) thr
         if (election == null) {
             throw new NotFoundException("Election for specified id does not exist");
         }
+        String darId = election.getReferenceId();
+        DataAccessRequest dar = Objects.nonNull(darId) ? dataAccessRequestService.findByReferenceId(darId) : null;
+        if (dar == null) {
+            throw new NotFoundException("Data Access Request for specified id does not exist");
+        } 
+        Integer userId = dar.getUserId();
+        List<LibraryCard> libraryCards = Objects.nonNull(userId) ? libraryCardDAO.findLibraryCardsByUserId(userId) : new ArrayList<>();
```

Comment:
It's more idiomatic in Java to use Collections.emptyList() as opposed to a new ArrayList<>.